corehq/toggles.py

```diff
@@ -81,7 +80,7 @@ def randomness_percent(self):
         return "{:.0f}".format(self.randomness * 100)
 
     def _get_identifier(self, item):
-        return '{}:{}:{}'.format(self.namespace, self.slug, item)
+        return '{}:{}:{}'.format(self.namespaces, self.slug, item)
```

Comment:
Seems fine since its formatting is going to be consistent unless python changes the way it converts lists to strings